src/storage/storage.cc

```diff
@@ -67,8 +67,9 @@ StorageManager *CreateStorageManager(const Context &ctx, const char *context,
                                      int num_gpu_device, std::string *pStrategy) {
   const auto env_var = env_var_name(context, pool_type);
   const char *type = getenv(env_var.c_str());
-  if (type == nullptr)
-    type = "Naive";   // default pool
+  if (type == nullptr) {
+    type = "Round";   // default pool
```

Comment:
it's obvious that round can help speed up certain dynamic input workloads, but tends to oom more frequently. I suggest we much very cautious about changing the default to round, unless there's a good fallback for oom handling.

Reply:
I share the concern on this. This change will impact those static-size static-graph models that were at the boundary of GPU memory limit. How many of the current GluonCV models fall in this category?

Reply:
My hope is of course to provide a good out-of-the-box usage experience to mxnet users. From what I observed, there seems to be more models with dynamic shape inputs than static ones, and many of the static-shape models can still run in this setting, hence the proposal.